blog/templates/categories.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html'   %}
+
+{% block content %}
+{% if category_posts %}
+<h1>{{ cats }}</h1>
+<ul>
+{% for post in category_posts %}
+    <li><a href="{% url 'article-detail' post.pk %}">
+        {{ post.title }}</a>
+        - {{ post.author.first_name}}
+        {{ post.author.last_name }} -
+        {{ post.post_date }}
+        {% if user.is_authenticated %}
+            - <a href="{% url 'update_post' post.pk %}">(Edit)</a>
+            <a href="{% url 'delete_post' post.pk %}">(Delete)</a>
+        {% endif %}
+        <br/>
+        {{ post.body|slice:":200"|safe }}</li>
+{% endfor %}
+</ul>
+{% else%}
+    <h2>Sorry, this page does not exist...</h2>
```

Comment:
Investigate how working with errors tmpls

Reply:
ok, I'll do it